backend/src/test/resources/testResultUpload/test-results-upload-all-fields.csv

```diff
@@ -1,3 +1,3 @@
-1234,Doe,Jane,Jingleheimer,123 Main St,Suite 401,Birmingham,AL,35226,Jefferson,205-999-2800,1/20/1962,F,White,Not Hispanic or Latino,Eng,test@test.com,123,ID NOW,94534-5,Detected,12/20/2021 14:00,12/20/2021 14:00,12/20/2021 14:00,12/20/2021 14:00,12/20/2021 14:00,Nasal Swab,1013012657,Smith MD,John,Smitty,400 Main Street,Suite 401,Birmingham,AL,35228,205-888-2000,01D1058442,My Urgent Care,400 Main Street,Suite 401,Birmingham,AL,35228,205-888-2000,N,N,N,01/01/1901,N,22232009,N,N,My Urgent Care,400 Main Street,Suite 100,Birmingham,AL,35228,205-888-2000,Test Comment,C,94534-1,"M, F, trans man, TW, nonbinary"
-5678,Doe,John,Jingleheimer,333 Elm St,Suite 401,Hoover,AL,35244,Shelby,205-222-2000,11/1/2001,MALE,BLACK OR AFRICAN AMERICAN,NOT HISPANIC OR LATINO,Spa,myemail@email.com,05be96ef-949f-4f46-9c07-2675e8abf412,BD Veritor System for Rapid Detection of SARS-CoV-2*,94558-4,NOT DETECTED,12/21/2021 17:00,12/21/2021 17:00,12/21/2021 17:00,12/21/2021 17:00,12/21/2021 17:00,ANTERIOR NARES SWAB,1013012657,Smith MD,John,Smitty,400 Main Street,Suite 401,Birmingham,AL,35228,205-888-2000,01D1058442,My Urgent Care,400 Main Street,Suite 401,Birmingham,AL,35228,205-888-2000,NO,YES,Y,01/01/2020,Y,HOSTEL,NO,NO,My Urgent Care,400 Main Street,Suite 100,Birmingham,AL,35228,205-888-2000,testy mctesterson,F,94534-2,"Male, NB"
+patient_id,patient_last_name,patient_first_name,patient_middle_name,patient_street,patient_street2,patient_city,patient_state,patient_zip_code,patient_county,patient_phone_number,patient_dob,patient_gender,patient_race,patient_ethnicity,patient_preferred_language,patient_email,accession_number,equipment_model_name,test_performed_code,test_result,order_test_date,specimen_collection_date,testing_lab_specimen_received_date,test_result_date,date_result_released,specimen_type,ordering_provider_id,ordering_provider_last_name,ordering_provider_first_name,ordering_provider_middle_name,ordering_provider_street,ordering_provider_street2,ordering_provider_city,ordering_provider_state,ordering_provider_zip_code,ordering_provider_phone_number,testing_lab_clia,testing_lab_name,testing_lab_street,testing_lab_street2,testing_lab_city,testing_lab_state,testing_lab_zip_code,testing_lab_phone_number,pregnant,employed_in_healthcare,symptomatic_for_disease,illness_onset_date,resident_congregate_setting,residence_type,hospitalized,icu,ordering_facility_name,ordering_facility_street,ordering_facility_street2,ordering_facility_city,ordering_facility_state,ordering_facility_zip_code,ordering_facility_phone_number,comment,test_result_status,test_ordered_code,genders_of_sexual_partners,syphilis_history
```

Comment:
⚠️ change here is adding a `syphilis_history` column that is empty for each of the two rows